src/gluonts/zebras/schema.py

```diff
@@ -29,27 +29,90 @@
 This module provides tooling to extract ``zebras.TimeFrame`` and
 ``zebras.SplitFrame`` instances from Python dictionaries::
 
-    schema = zebras.Schema(...)
+    schema = zebras.schema.Schema(...)
     tf = schema.load_timeframe(raw_dict)
 
     sf = schema.load_splitframe(raw_dict)
 
-The idea is to define expected types and shapes for each field (column) using
-``zebras.Field`` and then wrap them in a ``zebras.Schema``:
+The idea is to define expected types and shapes for each field (column) and
+then wrap them in a ``zebras.Schema``:
```

Comment:
Why `and them write them`?

```suggestion
The idea is to define expected types and shapes for each field (column)
in a ``zebras.schema.Schema``:
```